lfucg-exactions/server/accounts/viewsets.py

```diff
@@ -7,25 +9,36 @@
 from .serializers import *
 from .permissions import CanAdminister
 
+from django.conf import settings
+
+
 class AccountViewSet(viewsets.ModelViewSet):
     serializer_class = AccountSerializer
     queryset = Account.objects.all()
     permission_classes = (CanAdminister,)
 
     def get_queryset(self):
         queryset = Account.objects.all()
-
+        PageNumberPagination.page_size = 0
+        
+        paginatePage = self.request.query_params.get('paginatePage', None)
         query_text = self.request.query_params.get('query', None)
+
         if query_text is not None:
+            paginatePage = True
             query_text = query_text.lower()
-
             queryset = queryset.filter(
                 Q(account_name__icontains=query_text) |
                 Q(contact_full_name__icontains=query_text) |
                 Q(address_full=query_text) |
                 Q(phone__icontains=query_text) |
                 Q(email__icontains=query_text))
 
+
+        if paginatePage is not None:
```

Comment:
it doesn't matter if the query string says paginatePage=true, paginatePage=false or paginatePage=foo, you're still going to paginate the page.

Reply:
Do you have any suggestions about what to do here? I'm struggling to find an answer to what happens when you clear out the search input which fires through the `if query_text is not None`. If I take the `paginatePage = True` out of that it returns me all the results. 

I could set the parameter in the front end apiActions.js file as `/account/?query=${query}&paginatePage` ??